demo/demo_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "testing"
+
+func TestLocalOrchestration(t *testing.T) {
```

Comment:
This doesn't stop the whole network and doesn't do a resharing correct ?

Reply:
I understand the demo with processes still test that - I'm just raising the point if we should do it there or not as well - could increase coverage as well.

Reply:
I wanted to start with this and not delay unit tests more. I think this largely tests that the driver works enough to drive the local node, which is the main thing i want to not break as subsequent changes occur to the code base.

Reply:
for reference: this test takes less than a minute in CI, compared to ~3 minutes for the demo run